serverless/sql-databases/concepts.mdx

```diff
@@ -23,7 +23,80 @@ categories:
+</Concept>
+
+<Concept opened>
+## Cold start
+
+Cold start is the time an idle database takes to handle the first request. It usually takes less than five seconds.
+</Concept>
+
+<Concept>
+## Connection pooling
+
+Connection pooling is the process of mutualizing client connections used to connect to the database. By default, connections to PostgreSQL databases uses a lot of memory, which can have a significant impact on performance when hundreds of clients are connected to a database at the same time. With connection pooling, a common component (a "connection pooler", such as PgBouncer) connects to the database and maintains multiple connections to it. Clients then connect to the connection pooler instead of connecting directly to the database, allowing for better performance and more concurrent connections.  
+</Concept>
+
+<Concept>
+## Credentials
+
+Credentials are the login and password (also called "username" and "password" in PostgreSQL vocabulary) used by a user or an application to connect to a database. With Scaleway Serverless SQL Databases, logins are IAM principal IDs and passwords are IAM secret keys.
+</Concept>
+
+<Concept>
+## Engine
+
+A database engine is the software that stores and retrieves your data from a database. Serverless SQL databases use the PostgreSQL engine.
+</Concept>
+
+<Concept>
+## High availability
+
+Databases can become unavailable due to several issues (host crashes, disk corruption, network unavailability, etc.). High Availability allows to ensure that databases stay available most of the time and recover quickly from issues so that applications relying on them remain available as well.
+</Concept>
+
+<Concept>
+## Idle
+
+A database switches to an idle state if it receives no query for 5 minutes. You are not billed for compute resources while your database is in an idle state.
+</Concept>
+
+<Concept>
+## Logs
+
+Logs register the activity of your database. They provide useful information for debugging or to know more about the behavior and activity of your databases.
+</Concept>
+
+<Concept>
+## PostgreSQL
+
+PostgreSQL is an open source object-relational database system known for its reliability, robustness, and performance. 
+</Concept>
+
+<Concept>
+## Region and AZ
+
+<Macro id="region-and-az" />
+
+</Concept>
+
+<Concept>
+## Relational database
+
+A database type that relies on the relational model and that can be queried using the SQL language. In the relational model, different data objects are stored in different tables, which can be linked by common IDs, or keys. Relational databases provide an intuitive, straightforward way of representing data in tables, which favors data normalization and provide the strongest data consistency guarantees (compared to NoSQL alternatives).
```

Comment:
`Relational databases **provide** an intuitive, straightforward way of representing data in tables, which favors data normalization, and **provide** the strongest data consistency guarantees (compared to NoSQL alternatives).`

ou 

`Relational databases provide an intuitive, straightforward way of representing data in tables, which **favors** data normalization and **provides** the strongest data consistency guarantees (compared to NoSQL alternatives).`